lxd/main_init.go

```diff
@@ -249,9 +253,10 @@ func cmdInit() error {
 			}
 		}
 
-		if storageSetup && storageBackend == "zfs" {
+		if storageSetup && storageBackend != "dir" {
```

Comment:
So what's happening with dir exactly? Does the user just not get asked for pool name, ... if it's a dir backend?

If so, does that mean that they can't use "lxd init" to define a second dir pool?